css/src/components/segmented-control.scss

```diff
@@ -0,0 +1,131 @@
+$sc-padding-sm: 0.5em 1em !default;
+$sc-padding: 0.75em 1.25em !default;
+$sc-padding-lg: 1em 1.5em !default;
+
+$sc-font-color: $primary !default;
+$sc-border: $border-width solid $border !default;
+$sc-hover-background-color: $primary-background !default;
+$sc-hover-border: $border-width solid $border !default;
+$sc-hover-border-color: $primary-hover !default;
+$sc-active-hover-background-color: $primary-hover !default;
+$sc-active-hover-border-color: $primary-hover !default;
+$sc-active-background-color: $primary !default;
+$sc-active-border-color: $primary !default;
+$sc-active-color: $text-invert !default;
+
+.segmented-controls {
+	display: grid;
+	justify-content: center;
+	width: 100%;
+	grid-gap: $spacer-5;
+	grid-template-columns: max-content auto max-content;
+	font-size: $font-size-7;
+	white-space: nowrap;
+
+	&.segmented-lg {
+		font-size: $font-size-5;
+	}
+}
+
+.segmented-control-previous,
```

Comment:
Nit: all the sub-components should be placed withing the `segmented-controls` so they're not mis-used outside the component

```
.segmented-controls {
  ...

  .segmented-control-previous,
  .segmented-control-next {
    ...
  }
 
  ...
}

Reply:
This should be fixed now.